app/views/enquiries/_form.html.erb

```diff
@@ -1,8 +1,177 @@
+    <%= f.text_field :organisation_id, class: "form-control" %>
+  </div>
+    
+  <div class="form-group row">
+    <div class="col-sm-6"><%= f.label :client_used_before, "Client used service before?" %>
+    </div>
+    <div class="col-sm-14">
+      <div class="form-check">
+        <%= f.check_box :client_used_before, class: "form-check-input" %>
+      </div>
+    </div>
+  </div>
+  
+  <div class="form-row">  
+    <%= f.label :referred_by, "Referral" %>
+    <%= f.text_field :referred_by, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
     <%= f.label :first_name, "First Name" %>
     <%= f.text_field :first_name, class: "form-control" %>
   </div>
+    
+  <div class="form-row"> 
+    <%= f.label :second_name, "Last Name" %>
+    <%= f.text_field :last_name, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :telephone, "Tel No" %>
+    <%= f.text_field :telephone, class:   "form-control" %>
+  </div>
+  
+  <div class="form-group row">
+    <div class="col-sm-6"><%= f.label :client_used_before, "Contact via Voicemail?" %>
+    </div>
+    <div class="col-sm-14">
+      <div class="form-check">
+        <%= f.check_box :contact_via_voicemail, class: "form-check-input" %>
+      </div>
+    </div>
+  </div>
+  
+  <div class="form-group row">
+    <div class="col-sm-6"><%= f.label :client_used_before, "Contact via Text?" %>
+    </div>
+    <div class="col-sm-14">
+      <div class="form-check">
+        <%= f.check_box :contact_via_text, class: "form-check-input" %>
+      </div>
+    </div>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :email, "First Name" %>
+    <%= f.text_field :email, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :date_of_birth, "Date of Birth" %>
+    <%= f.text_field :date_of_birth, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_1, "Address Line 1" %>
+    <%= f.text_field :address_1, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_2, "Address Line 2" %>
+    <%= f.text_field :address_2, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_3, "Address Line 3" %>
+    <%= f.text_field :address_3, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_4, "Address Line 4" %>
+    <%= f.text_field :address_4, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_5, "Address Line 5" %>
+    <%= f.text_field :address_5, class: "form-control" %>
+  </div>
+    
+  <div class="form-row"> 
+    <%= f.label :address_6, "Address Line 6" %>
+    <%= f.text_field :address_6, class: "form-control" %>
+  </div>
+      
+  <div class="form-row">
+    <%= f.label :issue_status, "Issue Status" %>
+    <% issue_statuses = Enquiry.issue_statuses %>
+    <%= f.select(:issue_status, Enquiry.issue_statuses.keys) %>
```

Comment:
Nice, lots of these will need tables, and you'll be surprised how quickly this can be done in comparison to what's been done here. But this is an interesting approach, for when we're listing values which would never be dynamic.